tools/report.sh

```diff
@@ -191,18 +210,12 @@ for pod in $pods; do
 	  $platform logs $pod -p -c topic-operator -n $namespace 2>/dev/null > $direct/reports/podLogs/previous-"$pod"-topic-operator.log
 	  $platform logs $pod -p -c user-operator -n $namespace 2>/dev/null > $direct/reports/podLogs/previous-"$pod"-user-operator.log
 	elif [[ $pod =~ .*-kafka-[0-9]+ ]]; then
-	  $platform logs $pod -c tls-sidecar -n $namespace > $direct/reports/podLogs/"$pod"-tls-sidecar.log
```

Comment:
Please keep these for the time being since this is not versioned and might be used even with older versions of Strimzi which have the sidecars.